.travis.yml

```diff
@@ -13,6 +13,8 @@ env:
   - RABBITMQ_USERNAME="guest"
   - RABBITMQ_PASSWORD="guest"
   jobs:
+  - RABBITMQ_VERSION=3.10
```

Comment:
Actually Travis is not used anymore, I should remove this file :man_facepalming: 
I removed it on `master` so you can merge it in your branch